pom.xml

```diff
@@ -249,7 +249,7 @@
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>2.8.1</version>
                     <configuration>
-                        <argLine>-Duser.timezone=UTC</argLine>
+                        <argLine>-Duser.timezone=UTC -Dddf.version=${ddf.version}</argLine>
```

Comment:
Just a heads up, there will be a conflict here if https://github.com/codice/alliance/pull/149 is merged first.